history/parser.go

```diff
@@ -21,6 +21,8 @@ func PrintTopCommands(cfg config.Config, p Parser) {
 		log.Fatalf("Error parsing history: %v", err)
 	}
 
+	log.Printf("commands in PrintTopCommands: %#v", commands)
+
```

Comment:
Could we remove this log statement? When I create a release it looks like Printf logs end up in the binary.

Reply:
My bad! I thought I had removed all my debugging logs, but I guess I missed a couple.

I'll remove those and add that test case, then I'll circle back to you. 😁